sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -84,3 +84,7 @@
 --    desc: find all MyISAM tables and convert them to InnoDB.
 --    arguments: none
 --    behavior: can take a long time.
+
+#IfNotColumnType form_misc_billing_options icn_resubmission_number VARCHAR(35)
+ALTER TABLE form_misc_billing_options CHANGE `icn_resubmission_number` `icn_resubmission_number` VARCHAR(35) DEFAULT NULL;
```

Comment:
@stephenwaite ,
Does this also need to be modified in database.sql?

Reply:
good pt, thank you

Reply:
oh, it's already set there just fine, should i fix https://github.com/openemr/openemr/blob/master/sql/4_2_2-to-5_0_0_upgrade.sql

Reply:
why not? it can only help...

Reply:
This is odd. the database.sql should not be different. To fix this, recommend migrating these 3 lines to the end of the 4_2_2-to-5_0_0_upgrade.sql file
(and then this will go out in OpenEMR 5.0.1)